seatunnel-dist/pom.xml

```diff
@@ -68,6 +73,112 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>1.8</version>
+                <executions>
+                    <execution>
+                        <id>postinstall_script</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <phase>generate-resources</phase>
+                        <configuration>
+                            <target>
+                                <echo append="false" file="${project.build.directory}/postinstall.sh">#!/bin/sh
+                                    mkdir -p ${rpm.stack.root}/current
+                                    ln -sfn ${rpm.stack.root}/${rpm.version}/seatunnel ${rpm.stack.root}/current
+                                    chmod 755 ${rpm.stack.root}/${rpm.version}/seatunnel/bin/*.sh
+                                    chmod 755 /etc/seatunnel/conf/*.sh</echo>
+                            </target>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>postremove_script</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <phase>generate-resources</phase>
+                        <configuration>
+                            <target>
+                                <echo append="false" file="${project.build.directory}/postremove.sh">#!/bin/sh
+                                    rm -rf ${rpm.stack.root}/current/seatunnel
+                                    rm -rf ${rpm.stack.root}/${rpm.version}</echo>
+                            </target>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>rpm-maven-plugin</artifactId>
```

Comment:
Some question:
1. Do we need users to manually build rpm packages at present?
2. Can the corresponding documentation be added?
3. Can we push rpm to some third-party repositories in the future? For example, EPEL?